account.go

```diff
@@ -1,6 +1,8 @@
 package fc
 
 import (
+	"encoding/json"
+	"net/http"
```

Comment:
If you want your ci to run through, instead of using fork, just create the branch and send the mr

Reply:
I have been a bit busy on this, let me take a look this weekend